README.md

```diff
@@ -86,3 +86,8 @@ Note that when the debugger first activates the user is always looking at transl
 ## Namespacing
 
 Apostrophe has basic support for separate namespaces of i18n keys. However, because of the impact on the layout of the display, the namespace associated with a key is not displayed by the debugger. Generally it is clear from context, for instance anything in the Apostrophe admin UI is likely to be in the `apostrophe` namespace.
+
+## Permissions
+
+By default, only users from the `admin` group have access to this menu in the admin bar. If another group needs it, it is configurable in the `Groups` entry in the menu bar under `I18n Debugger`.
```

Comment:
This sentence is correct, but it made me think you were suggesting that permissions were configured by clicking "I18n Debugger" in the admin bar.

How about this:

"By default, only users with the site-wide `admin` permission have access to the i18n debugger. If another group needs access, you can edit the group, click on the 'Permissions' tab, and check the 'i18n debugger' box. If you do not see 'groups' on your admin bar, see the apostrophe permissions documentation for how to activate editable groups."

Reply:
Great, thank you